inverse_covariance/inverse_covariance.py

```diff
@@ -12,6 +13,9 @@ def _init_coefs(X, method='corrcoef'):
     elif method == 'cov':
         init_cov = np.cov(X, rowvar=False)
         return init_cov, np.max(np.abs(np.triu(init_cov)))
+    elif method == 'spearman':
```

Comment:
add comments/documentation to the classes that ultimately use this initialization, there are a few in `quic_graph_lasso.py`, e.g., https://github.com/skggm/skggm/blob/develop/inverse_covariance/quic_graph_lasso.py#L204-L207